.mergify.yml

```
@@ -55,7 +55,17 @@ pull_request_rules:
             git merge upstream/{{base}}
             git push upstream {{head}}
             ```
-  - name: Automatic squash and merge with success checks and the files matching the regex ^testing/environments/snapshot* are modified.
+  - name: automatic approval for mergify pull requests with bump updates
+    conditions:
+      - author~=^mergify(|-preview)\[bot\]$
```

author is `mergify[bot]` , so I'll remove the `preview` and use text-match instead a regex

```suggestion
      - author=mergify[bot]
```